source/gui/settingsDialogs.py

```diff
@@ -1161,18 +1130,10 @@ def onOk(self, evt):
 			# The list of synths has not been populated yet, so we didn't change anything in this panel
 			return
 
-		config.conf["speech"]["outputDevice"]=self.deviceList.GetStringSelection()
 		newSynth=self.synthNames[self.synthList.GetSelection()]
 		if not setSynth(newSynth):
-			# Translators: This message is presented when
-			# NVDA is unable to load the selected
-			# synthesizer.
-			gui.messageBox(_("Could not load the %s synthesizer.")%newSynth,_("Synthesizer Error"),wx.OK|wx.ICON_WARNING,self)
+			_synthWarningDialog(newSynth.name)

```

Comment:
This causes the following traceback:

```
Traceback (most recent call last):
  File "gui\settingsDialogs.pyc", line 1135, in onOk
AttributeError: 'str' object has no attribute 'name'
```

Reply:
Fixed in 1e6ee08e0c7f179f3dc38df706f0f5a3fd5b5edf